runtime/gc_glue_java/PointerArrayObjectScanner.hpp

```diff
@@ -84,14 +84,10 @@ class GC_PointerArrayObjectScanner : public GC_IndexableObjectScanner
 	 * @param[in] startIndex The index of the first element to scan
 	 */
 	MMINLINE static GC_PointerArrayObjectScanner *
-	newInstance(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, void *allocSpace, uintptr_t flags, uintptr_t splitAmount, uintptr_t startIndex = 0)
+	newInstance(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, fomrobject_t *basePtr, void *allocSpace, uintptr_t flags, uintptr_t splitAmount, uintptr_t sizeInElements,  uintptr_t startIndex = 0)
```

Comment:
I have a feeling that we don't need sizeInElements.

If split flag is not set, splitAmount and sizeInElements should have the same value.

Reply:
Or I should say, we don't need splitAmount anymore - just rename splitAmount to sizeInElements, and document that if:
1) splitting is used, it's the total size of array
2) splitting is not used, it's the rest to be scanned.